tests/reals/CMakeLists.txt

```diff
@@ -3,15 +3,13 @@
 
 cmake_minimum_required(VERSION 2.8.11)
 
-if(MSVC)
 set(REAL_SM_C_FILES
     real_sm.c
 )
 set(REAL_SM_H_FILES
     real_sm.h
     real_sm_renames.h
 )
```

Comment:
I think we should just dump these in the main lists below now, no need for the separate variable